src/components/project/sections/the-challenge.tsx

```diff
@@ -1,27 +1,222 @@
+  line-height: 1.15;
+  letter-spacing: 2px;
+  font-weight: 500;
+`;
+
+const StyledMobileParagraph = styled.p`
+  font-size: ${getRemSize(dimensions.textSizes.normal.desktop)};
+  line-height: 1.3;
+  letter-spacing: 1px;
+  font-weight: 400;
+  text-indent: 72px;
+  margin-bottom: 40px;
+`;
+const StyledMobileParagraphImage = styled.span`
+  font-size: ${getRemSize(dimensions.textSizes.normal.desktop)};
+`;
+
+const variants = {
+  open: {
+    y: 0,
+    opacity: 1,
+    transition: {
+      y: { stiffness: 1000, velocity: -100 },
+    },
+  },
+  closed: {
+    y: 50,
+    opacity: 0,
+    transition: {
+      y: { stiffness: 1000 },
+    },
+  },
+};
+
 export default function DynamicTextAndImages({ content }: IProps) {
+  const isDesktop = useIsDesktop();
+  const isTablet = useIsTablet();
+  const [hoverIndex, setHoverIndex] = useState(null);
+
+  const onHover = (index) => {
+    setHoverIndex(index);
+  };
+
+  const onHoverOver = () => {
+    setHoverIndex(null);
+  };
+
   return (
-    <>
-      {content.map((item, index) => {
-        <p>{item.text}</p>;
-        {
-          item.image && (
-            <Image
-              key={index}
-              width={500}
-              height={200}
-              alt={`Gallery Image ${index}`}
-              src={item.image.node.sourceUrl}
-            />
-          );
-        }
-      })}
-    </>
+    <StyledBigWrapper>
+      <Row>
+        {isDesktop ? (
+          <>
+            <Col start={1} span={4}>
+              <StyledPillWrapper>
+                <Pill pillText="The challenge" />
+              </StyledPillWrapper>
+            </Col>
+            <Col start={5} span={8}>
+              <StyledWrapper>
+                <StyledParagraph>
+                  {content.map((item, index) => {
+                    return item?.image ? (
+                      <StyledParagraphImage
+                        onMouseEnter={(e) => onHover(index)}
+                        onMouseLeave={(e) => onHoverOver()}
+                      >
+                        {" "}
+                        <span>{item.text}</span>
+                      </StyledParagraphImage>
+                    ) : (
+                      <>{item.text}</>
+                    );
+                  })}
+                </StyledParagraph>
+                {hoverIndex && (
+                  <StyledImage
+                    onMouseEnter={(e) => onHover(hoverIndex)}
+                    onMouseLeave={(e) => onHoverOver()}
+                    variants={variants}
+                    initial="closed"
+                    animate={hoverIndex ? "open" : "closed"}
+                    // initial={{ opacity: 0, scale: 0.5 }}
+                    // animate={{ opacity: 1, scale: 1 }}
+                    // transition={{
+                    //   duration: 0.5,
+                    //   delay: 0.2,
+                    //   ease: [0, 0.71, 0.2, 1.01],
+                    // }}
+                    width={375}
```

Comment:
If you add these functions in here too, it prevents a bug where when the mouse hovers over the text and the image on top of it, it glitches out a bit

```suggestion
                    width={375}
                    onMouseEnter={(e) => onHover(hoverIndex)}
                    onMouseLeave={(e) => onHoverOver()}
```